R/dvhydrograph-render.R

```diff
@@ -293,6 +295,9 @@ getDVHydrographPlotConfig <- function(plotItem, plotItemName, yLabel="", maxIvLa
     groundWaterLevels = list(
       points = append(list(x=x, y=y, legend.name="Measured Water Level (GWSI)"), styles$gw_level_points)
     ),
+    waterQualityData = list(
```

Comment:
Yeah NULLs are handled by the plotItem function and if the item to be plotted is null it is skipped and the fetch style function is never called.